src/actions/TemperatureSyncActions.js

```diff
@@ -104,31 +104,30 @@ const downloadLogsComplete = () => ({
   type: TEMPERATURE_SYNC_ACTIONS.DOWNLOAD_LOGS_COMPLETE,
 });
 
+const saveLogs = (logData, sensor) => async dispatch => {
+  dispatch(startSavingTemperatureLogs(logData?.length));
+  const timeNow = new Date().getTime();
+
+  UIDatabase.write(() =>
+    logData.forEach(({ temperature }, i) => {
+      const timestamp = timeNow - (i + 1) * MILLISECONDS.FIVE_MINUTES;
```

Comment:
Nit: I think specifying the base unit of milliseconds in the constant is a bit redundant since the key specifies minutes? Could have something like `5 * TIME.MINUTE` to make it a bit more flexible.